assembly/std/address.ts

```diff
@@ -1,45 +1,57 @@
 import { Valider, Serializable, Args, Result } from '@massalabs/as-types';
 
 /**
- * A Massa's blockchain address.
- *
+ * Represents a Massa blockchain address.
  */
 export class Address implements Valider, Serializable {
   /**
    * Creates a new Address;
    *
-   * @param bs - Byte string.
-   * @param isValid - default true
+   * @param _value - A byte string that represents the address data.
+   * @param _isValid - Defaults to `true`.
    */
   constructor(private _value: string = '', private _isValid: bool = true) {}
 
   /**
-   * Returns if the Address is still valid.
+   * Returns whether the Address is still valid or not.
    *
-   * see https://github.com/massalabs/massa-sc-runtime/issues/142
+   * @returns A boolean value indicating whether the address is still valid.
+   *
+   * @see https://github.com/massalabs/massa-sc-runtime/issues/142
```

Comment:
Please can you reference this one instead ?
https://github.com/massalabs/massa-as-sdk/issues/195
```suggestion
   * @see https://github.com/massalabs/massa-as-sdk/issues/195
```